testing/simapp/app.go

```diff
@@ -515,11 +515,12 @@ func NewSimApp(
 	app.IBCKeeper.SetRouter(ibcRouter)
 
 	clientRouter := app.IBCKeeper.ClientKeeper.GetRouter()
+	storeProvider := ibcclienttypes.NewStoreProvider(keys[ibcexported.StoreKey])
```

Comment:
If people feel it would be nicer to access this from the client keeper we could add the following, since the client keeper maintains a ref to the ibc store key:

This a bit of a dumber approach which means app devs also have to not worry about what it means to create this type with some particular key (ibc).

```suggestion
	storeProvider := app.IBCKeeper.ClientKeeper.GetStoreProvider()
```

Would probs vote for this myself!

Reply:
Yeah, this makes sense to me too